app/main/views/briefs.py

```diff
@@ -148,7 +150,7 @@ def list_briefs():
         )
 
         return jsonify(
-            briefs=[brief.serialize() for brief in briefs.items],
+            briefs=[brief.serialize(with_users) for brief in briefs.items],
```

Comment:
Super-duper nitpick (I know it's already been merged), but I think using a keyword argument parameter might be a tiny bit clearer here.  

ie, given that `with_users` is `True` or `False`, I think that calling `brief.serialize(True)` seems less intuitive than `brief.serialize(with_users=True)`. (Even though I get that what it would actually look like is `brief.serialize(with_users=with_users)`.)

Really not a big deal.